src/memory/AmdFlash.cc

```diff
@@ -563,6 +573,18 @@ void AmdFlash::write(size_t address, uint8_t value, EmuTime::param /*time*/)
 		} else {
 			cmd.clear();
 		}
+	} else if (state == State::ERASE_SECTOR) {
+		if (checkCommandEraseAdditionalSector(time)) {
+			// do nothing, we're still matching a command, but it is not complete yet
+		} else {
+			cmd.clear();
+		}
+	} else if (state == State::ERASE_CHIP) {
+		if (false) {
```

Comment:
This `if (false)` is unexpected (and it's still like this in the final code).
Is this an oversight? If not then maybe add an extra comment for why it's written like this.

Reply:
Just didn’t want to break the pattern of `if (checkCommandX()) { /* do nothing */ } else { cmd.clear }`.